backend/api/views.py

```diff
@@ -348,11 +348,15 @@ def purchase_items(self, request, pk=None):
                 "item_id"
             ]
             item = Item.objects.get(pk=int(item_id))
+            if((item.bulkMinimum <= i["quantity"]) and (item.bulkMinimum > 0)):
```

Comment:
@ceguiluzrosas @MattKersey 
I'm doing this bulkminimum > 0 here because I think the default value for "not a bulk order" is 0. is there any way we can get default to None if its not a bulk item, then in the Serializer have it be a javascript null? That way I can remove this check on the endpoint and the FE. Maybe not a priority

Reply:
We can create a validator within the class to ensure that price/stock/whatever can be either None or some positive value. We will need to eliminate some parameters such as max_digits and decimal places in case the value is None. @jkim10  

```python
class Item(models.Model):

    def customValidator(self, value):
        if value is not None and value < 0.0:
            raise ValidationError("Value must either be null or positive")
        elif value is not None and len(str(value))>13:
            raise ValidationError("Value must not have more than 12 digits")
        elif value is not None and len(str(value).rsplit('.')[-1]) > 2:
            raise ValidationError("Value must not have more than 2 decimal places")

   attribute = models.DecimalField(
        null=True,
        blank=True,
        validators=[customValidator],
    )
```
Note that this would require editing a couple tests.

Reply:
hmmm maybe we just leave it as it is for now then since its working. if we have time, we can do that